virttest/utils_misc.py

```diff
@@ -2475,11 +2480,15 @@ def get_win_disk_vol(session, condition="VolumeName='WIN_UTILS'"):
     Getting logicaldisk drive letter in windows guest.
 
     :param session: session Object.
-    :param condition: supported condition via cmd "wmic logicaldisk list".
+    :param condition: supported condition via powershell cmd "Get-CimInstance Win32_LogicalDisk".
 
     :return: volume ID.
     """
-    cmd = "wmic logicaldisk where (%s) get DeviceID" % condition
+    c_name, c_value = condition.split("=")
```

Comment:
@leidwang this logic seems to be incorrect. In the balloon_hotplug variants I am hitting the following:
```
[stdlog] 2025-03-03 15:09:30,148 avocado.test stacktrace       L0050 ERROR|   File "/usr/local/lib/python3.9/site-packages/avocado_framework_plugin_vt-109.0-py3.9.egg/virttest/utils_misc.py", line 2482, in get_win_disk_vol
[stdlog] 2025-03-03 15:09:30,148 avocado.test stacktrace       L0050 ERROR|     c_name, c_value = condition.split("=")
[stdlog] 2025-03-03 15:09:30,148 avocado.test stacktrace       L0050 ERROR| ValueError: not enough values to unpack (expected 2, got 1)
```
Which leads to:
`avocado.core.exceptions.TestError: Could not get virtio-win disk vol!`

Reply:
Same for balloon_service variants

Reply:
Could you add the full debug.log to issue3268, thanks.

Reply:
@leidwang sorry for the late reply, debug.log updated !